engine/src/main/resources/assets/ui/onlinePlayersOverlay.ui

```diff
@@ -19,7 +19,7 @@
                     "contents": [
                         {
                             "type": "UILabel",
-                            "text": "Online Players",
+                            "text": "${engine:menu#online-players},
```

Comment:
Missing terminating `"` here but fixing as part of merge - trivial :-)

Protip: Tell IntelliJ to treat the .ui extension (and prefab etc) as JSON for handy syntax highlighting.